packages/hiro-graph-client/src/eventstream.js

```diff
@@ -329,12 +329,23 @@ export default class EventStream {
                 );
 
                 if (this._scopeId) {
-                    ws.send(
-                        JSON.stringify({
-                            type: 'subscribe',
-                            id: this._scopeId,
-                        }),
-                    );
+                    if (typeof this._scopeId === 'string') {
+                        ws.send(
+                            JSON.stringify({
+                                type: 'subscribe',
+                                id: this._scopeId,
+                            }),
+                        );
+                    } else if (Array.isArray(this._scopeId)) {
+                        this._scopeId.forEach((id) =>
+                            ws.send(
+                                JSON.stringify({
+                                    type: 'subscribe',
+                                    id,
+                                }),
+                            ),
+                        );
+                    }
```

Comment:
Actually we assume, that this._scopeId presents (condition `if (this._scopeId)` outside), and we can only check it's an array or not. In this case a number of if conditions will be reduced:)

Reply:
Thanks! I made a refactoring of scopeId type check. Please, have a look